docs/source/hardware/hardware_overview.md

```diff
@@ -66,3 +64,90 @@ You can add these same flags as an additional line in your `sbatch` script via (
 :::{note}
 Using the --constraint flag can mean that you will wait longer for your job to start, as the {term}`scheduler` ({term}`Slurm`) will need to find and allocate the appropriate hardware that you have specified for your job. For more information about running jobs, see {ref}`using-slurm`. Finally, at this time only the OR operator `|` is supported when using `--constraint`.
 :::
+
+
+## GPU Hardware
+
+Table 2 below shows the GPU Types, Architecture, memory and other features of the GPUs available on the HPC. All of these are NVIDIA GPUs available on gpu-equipped partitions.
+
+(gpu-table)=
+:::{list-table} The NVIDIA GPUs available on gpu-equipped partitions.
+--------------
+header-rows: 1
+align: center
+widths: auto
+--------------
+
+* - GPU Type
+  - GPU Architecture
+  - Memory (GB)
+  - Tensor Cores
+  - CUDA Cores
+  - Public Nodes (*x* # GPUs)
+  - Private Nodes (*x* # GPUs)
+* - [P100](https://northeastern.zoom.us/s/92041124566)
```

Comment:
Should we remove the P100s from the table because they are only in the courses-gpu partition now?

Reply:
Issue #264

Reply:
Also, not sure how the zoom link appeared as a link to P100s. Removing P100 from this table.